osbuild/util/parsing.py

```diff
@@ -82,6 +82,14 @@ def parse_location_into_parts(location: str, args: Dict) -> Tuple[str, str]:
     path information as passed by osbuild.api.arguments()
     """
 
+    if "://" not in location:
+        print("WARNING: location has no scheme, assuming 'tree://'")
```

Comment:
perhaps `INFO:` here would be more appropriate since a lot of manifests are going to hit this today.

Reply:
Done